pytest/tests/mocknet/remote_node.py

```diff
@@ -98,8 +98,14 @@ def get_nodes(chain_id, start_height, unique_id):
         sys.exit(f'no known nodes matching {pattern}')
 
     traffic_generator = None
+    tracing_server = None
+    state_dumper = None
     nodes = []
     for n in all_nodes:
+        if n.instance_name.endswith('tracing-server'):
+            tracing_server = n
+        if n.instance_name.endswith('dumper'):
```

Comment:
wait shouldn't these branches be `elif`s here? because otherwise they'll be added to the `nodes` list below